Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureExpressApp/app.ts

```diff
@@ -1,10 +1,11 @@
-import debug = require('debug');
-import express = require('express');
-import path = require('path');
+import * as express from 'express';
+import { AddressInfo } from "net";
+import * as path from 'path';
 
 import routes from './routes/index';
 import users from './routes/user';
 
+const debug = require('debug')('my express app');
```

Comment:
What does the `('my express app')` do?  It doesn't seem to have been in the original.

Reply:
So [debug](https://www.npmjs.com/package/debug) doesn't immediately give you a logging function, it gives you a function that takes a string and *returns* a logging function that, when called, will write to std.err with that string as a tag:
![image](https://user-images.githubusercontent.com/38218905/100145591-65c99000-2e4d-11eb-999d-4aa2e6061754.png)

Previously, the template code was never `debug` with the initial tag string, so the output never actually got logged.

This should actually be updated in the JavaScript version of the template as well, now that I think of it.

Reply:
Wacky